### PR TITLE
✨ add resources

### PR DESCRIPTION
**Pull request by JuanPabloDiaz**
# Add icons to the resources DB

- RealFaviconGenerator
- Favicon Generator
- Feather Icons
- Boxicons
- Zondicons
- Iconscout
- FlatIcon
- Icon Icons
- React Icons

Issue #12

Be sure to complete and check the following steps:

- [x] My changes were made in the `source-web/src/assets/db` folder.
- [x] The resource(s) is completely or partially free.
- [x] The resource(s) is not already in the database.
- [x] All resources have an id, order, name, link and img.
- [x] The ids and urls of the images are not repeated.

### CODE REVIEW

**Comment by cosmoart** [comment]
Thank you for your interest in contributing, before accepting the pull request I have some suggestions:

- To make it easier to read the new resources, use the same indentation of the file (1 tab).
- For image ids, use more complete names, like "react-icons" or "icon-icons", not 51, 52.

As for "Favicon Generator" and "Real Favicon Generator" it would be better to add them in the "tools" category and deduce their "order" a lot as they are tools with a very circumstantial use.

File: source-web/src/assets/db/tools.json [hunk 1]
```diff
@@ -126,5 +126,23 @@
 		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/v1707069166/freesets/tools/hvxjb3ktzko8xztb1qcl.webp",
 		"link": "https://transform.tools/",
 		"tags": ["Transform data"]
+	},
+	{
+		"id": 17,
+		"order": 30,
+		"name": "Favicon Generator",
+		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/freesets/icons/favicon-generator",
```
Comment (cosmoart) [inline source-web/src/assets/db/tools.json:134]: The correct url would be: https://res.cloudinary.com/cosmocloudinary/image/upload/freesets/tools/favicon-generator

File: source-web/src/assets/db/tools.json [hunk 2]
```diff
@@ -126,5 +126,23 @@
...
+		"link": "https://www.favicon-generator.org/",
+		"license": "Free",
+		"tags": ["Favicon"]
+	},
+	{
+		"id": 18,
+		"order": 30,
+		"name": "Real Favicon Generator",
+		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/freesets/icons/real-favicon",
```
Comment (cosmoart) [inline source-web/src/assets/db/tools.json:143]: The correct url would be: https://res.cloudinary.com/cosmocloudinary/image/upload/freesets/tools/real-favicon

**Review by cosmoart — CHANGES_REQUESTED** [review]
Please use an indentation of 1 tab.

File: source-web/src/assets/db/icons.json [hunk 3]
```diff
@@ -1,517 +1,550 @@
...
+		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/freesets/icons/53",
+		"license": "Free",
+		"order": 26
+	},
+	{
+		"id": 54,
+		"name": "Iconscout",
+		"link": "https://iconscout.com/",
+		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/freesets/icons/54",
+		"license": "Free & Paid",
```
[the remark this answers is not in the document]
Comment (JuanPabloDiaz) [reply source-web/src/assets/db/icons.json]: If I'm not mistaken, the license for this product is [MIT](https://github.com/Iconscout/iconscout-plugins/blob/master/LICENSE) @cosmoart

**Comment by JuanPabloDiaz** [comment]
> Please use an indentation of 1 tab.

My VS code was changing the indention for those files when saving it. I changed my settings to allow only 1 tab indentation, but I don't think that changed anything on the actual project. (I do see the files formatted correctly in my VS code)

**Review by cosmoart — APPROVED** [review]

**Pull request merged**